src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -99,14 +99,24 @@ function runEventInBatch(event) {
 function startWatchingForChangeEventIE8(target, targetID) {
   activeElement = target;
   activeElementID = targetID;
-  activeElement.attachEvent('onchange', manualDispatchChangeEvent);
+
+  if (activeElement.attachEvent) {
+    activeElement.attachEvent('onchange', manualDispatchChangeEvent);
+  } else {
+    activeElement.addEventEventListener('change', manualDispatchChangeEvent, false);
```

Comment:
Can this actually be reached? It's kind of hard to tell at first glance, but this only targets IE8 right? So it shouldn't be.